crates/bevy_input_focus/src/lib.rs

```diff
@@ -149,12 +149,21 @@ pub struct InputDispatchPlugin;
 
 impl Plugin for InputDispatchPlugin {
     fn build(&self, app: &mut App) {
-        app.insert_resource(InputFocus(None))
+        app.add_systems(Startup, set_initial_focus)
+            .insert_resource(InputFocus(None))
             .insert_resource(InputFocusVisible(false))
             .add_systems(PreUpdate, dispatch_keyboard_input);
     }
 }
 
+/// Sets the initial focus to the primary window, if any.
+pub fn set_initial_focus(
+    mut input_focus: ResMut<InputFocus>,
+    window: Single<Entity, With<PrimaryWindow>>,
```

Comment:
I forget, does `Single` panic when there _isn't_ a single matching entity, or does this system just not run? I know there was some heated debate around the defaults here.

Reply:
System just won't run :)